### ISSUE TITLE
Does not find the recipient

### ISSUE THREAD

**Issue by Portugao** [zikula-modules/InterCom#105]
If I enter 2 characters, the loading icon rotates, but i get no result.
Do not know, if it is caused by my local install with wampp?!

**Comment by Portugao** [comment]
No, my local install is not the reason, just tested online; does not work.
Zikula 1.5.2
Php 5.6

**Comment by Portugao** [comment]
Strange, new try and it works.

**Comment by Kaik** [comment]
Well, I have tested this many times and it is hard to say what might be the reason for this behavior. It is an ajax call so things might happen. We need to test this as much as possible and optimise it.

**Comment by Guite** [comment]
Does InterCom use the user auto completion form type offered by the core or a custom implementation?

**Comment by Kaik** [comment]
I think it's custom because it is made to select users and groups not only users.
